### DISCUSSION TITLE
2024-04-09、2024-04-10 周题库讨论

### DISCUSSION THREAD

**Discussion by xbn2002** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#173] [General]
#### MlgmXyysd 的 Telegram 频道
- https://t.me/MlgmXyysd_bibilailai
- 进入频道后，您还可通过以下方法加入**附属群聊**：
<table align='center'>
  <tr>
    <th>手机端</th>
    <th>电脑端</th>
  </tr>
  <tr>
    <td><img width='300' alt='9d72f92993ce6780da1d93dd4b282ff' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/223e9b12-d0b7-4560-981e-0006865ac708'><br><img width='300' alt='d57b66f3c417bd3c1bfd4804b3f3949' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/15dcb998-1f76-4cb1-8bed-e85ee1c521aa'></td>
    <td><img width='119' alt='ffb1d42c1b428d2483628c6fd535837' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/f8552112-af4e-4595-9968-f66e4f5c616d'></td>
  <tr>
</table>

#### 目录导航
- [2024-04-09 08:00-08:15](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/173#discussioncomment-9045817)
- [2024-04-09 14:45-15:00](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/173#discussioncomment-9045818)
- [2024-04-10 08:00-08:15](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/173#discussioncomment-9045819)

#### 2024-04-09、2024-04-10《米米 BL 解锁全国统一考试》
- **2024-04-09 8:00-8:15、14:45-15:00 和 2024-04-10 8:00-8:15，共有三场答题**
- **特别提醒：米米已经添加提示，填空题 请勿直接复制 解析答案，否则将清零答题分数！特别是 千万不要 把 `至少需要输入___个字符` 也复制进去！**
- 由于米米的答题时间开放时间极短，故不保证解析及时更新。

> ### 申请前必读
> 1. Bootloader解锁会导致设备面临各类安全风险，并对设备造成不可预估的影响，申请解锁代表您已经充分了解相关风险
> 2. 解锁后请勿将设备用于非正常用途，若检测到账号异常行为，可能导致账号被封禁
> 3. 本项目仅适合正在使用小米澎湃OS且设备处于上锁状态的用户报名，如果您的设备暂未升级到小米澎湃OS，您可以继续通过原有途径获取BL锁解锁权限
> 4. 为确保答题的公平性，提升答题体验，每周将安排多场答题时间（法定假期除外）。近期答题场次如下：**4月9日上午8:00 - 8:15、4月9日下午14:45 - 15:00、4月10日上午8:00 - 8:15**，您可以任选其中一个时段参与，**每次参与需间隔7天**，还请合理安排答题时间（**温馨提示：每场答题的最迟进场时间为该场次答题开始后的10分钟**）
> 5. **答题成绩每周一晚上23:59失效**，如答题通过，请务必在周一晚上23:59前完成资格申请，避免成绩失效影响申请
> 6. 报名成功不代表审核通过，解锁权限获取以最终审核结果为准；通过社区申请获得解锁权限后，若您绑定设备时该设备系统为小米澎湃OS，绑定成功后解锁等待期为72小时，请您知悉

- 答题链接：[一键直达](https://leijun.jinfan.online/blapply/)

> ### BL解锁申请资格答题测试
> 【填空题说明】填空题采用人工阅卷方式，人工阅卷预计在【下一答题周期开始前】完成。①您提交问卷时的实时成绩仅为【选择题得分】；②人工阅卷后，如您的【选择题+填空题总分】未达到分数线时，您的成绩将仅展示【选择题得分】；③人工阅卷后，如您的【选择题+填空题总分】达到分数线时，您的成绩将展示为【选择题+填空题总分】
> 【无法展示成绩的原因】①请务必通过“小米社区App-内测中心-BL解锁申请-去答题”进入本次答题界面并完成答题，未使用小米社区App进行答题将视为无效答题，无法展示成绩；②等级未达5段，无法展示成绩；③答题超时提交，无法展示成绩
> 请您充分知晓解锁和刷机存在的风险，并在可承担该风险的情况下申请Bootloader解锁
> 请您填写相关问题，为避免成绩无效，**建议作答时间不少于【5分钟】且不超过【15分钟】**，请合理安排答题时间。本期答题得分高于85分（不含85分）即可通过，感谢您的支持！

**Comment by xbn2002** [comment]
2024-04-09 08:00-08:15
- 1574504 **以下关于《小米账号使用协议》和《BL解锁申请资格答题测试》的相关要求，说法正确的有** 【多选题】
  - 5165527 填空题务必不要复制粘贴其他网站的答案，答案雷同或相似度高，将视为无效问卷，无法获得分数
    - ✔️ 选中。根据问卷填空题页面提示，如出现使用AI生成、复制粘贴、雷同试卷等情况，将视为无效作答，无法通过审核。
  - 5165528 如果小米发现您并非该账号初始申请注册人，小米有权在未经通知的情况下终止您使用该账号
    - ✔️ 选中。根据《小米账号使用协议》，任何情况下，您都无权使用由他人注册的账号。
  - 5165529 当您获得BL解锁资格后，通过其他途径参与本答题测试，会根据行为严重程度，面临取消资格、账号终止使用等处罚
    - ✔️ 选中。根据《BL解锁申请资格答题测试》的相关要求，您只能通过「小米社区」APP 参与答题。
  - 5165530 当您小米社区等级未达到5段及以上时，通过其他途径参与本答题测试，会根据行为严重程度，面临账号终止使用等处罚
    - ✔️ 选中。本选项的表述是对上一选项的补充。
- 1574505 **已满足BL解锁申请条件，但申请后被拒绝的原因，下列说法正确的有** 【多选题】
  - 5165531 在社交媒介恶意诋毁、谩骂、造谣等
    - ✔️ 选中。您不得在社交媒介对小米恶意诋毁、谩骂、造谣等。
  - 5165532 出现被判定为作弊的行为
    - ✔️ 选中。根据解锁申请详细说明，“代答”等账号异常行为属于违规，解锁申请不予通过，且账号将被封禁。
  - 5165533 不填写申请理由、或使用AI生成申请理由、或申请理由出现过辱骂威胁等内容
    - ✔️ 选中。您必须在申请时认真填写真实的申请理由。
  - 5165534 其他违反小米账号使用协议的相关情形
    - ✔️ 选中。您不得违反小米账号使用协议。
  - 5165535 确认无任何违规情形，被拒绝后可尝试再次申请
    - ✔️ 选中。如您无上述违规情形，在申请被拒绝后，可在 7 天后重新答题并申请。
- 1574506 **BL解锁面向发烧友和开发者开放，因此在资格筛选时，会考察是否符合发烧友的特质。下面关于发烧友的特质，说法正确的有** 【多选题】
  - 5165536 发烧友指的是对某一领域极度热爱，并对此领域的产品进行深入研究
    - ✔️ 选中。公告类型题，对上一题的正确选项做出了解释。
  - 5165537 发烧友具备探究精神，因此在遇到不了解的问题时，会在合理范围内主动探寻答案，但不会寻求不正当“捷径”或采用违规方式
    - ✔️ 选中。
  - 5165538 发烧友具备坚韧不拔的品质，因此在遇到失败或挫折时，会不断尝试和努力，但不会去造谣、诋毁
    - ✔️ 选中。
  - 5165539 发烧友具备学习精神，因此在遇到新鲜事物或新知识时，会乐于学习和钻研，不会肆意谩骂
    - ✔️ 选中。
  - 5165540 发烧友具备分享精神，因此在有一些好的知识或经验时，会积极分享和交流，不会去利用“资源”去“谋利”
    - ✔️ 选中。
- 1574507 **BL解锁危害多，BL解锁可能造成各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况，因此强烈建议您非必要情况不申请BL解锁。请问您是否已经知悉以上风险和危害，并愿意承担以上全部风险和危害？** 【单选题】
  - 5165541 本人愿意承担以上全部风险和危害，接受各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况
    - ✔️ 选中。免责声明，不做解释。
  - 5165542 本人不愿意承担以上全部风险和危害，立即关闭界面，停止答题和申请
    - ❌ 不选。
- 1574508 **下列说法正确的是** 【单选题】
  - 5165543 fastboot getvar all &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ❌ 不选。子命令 getvar 用于从 BootLoader 中读取手机信息。all 表示输出全部信息，包括 product, devices, anti 等字段信息。
  - 5165544 fastboot flash &lt;镜像路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ❌ 不选。根据 fastboot 使用说明，刷入（flash）子命令格式为 flash PARTITION [FILENAME]。
  - 5165545 fastboot flash &lt;分区名&gt; &lt;镜像文件名或路径&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ✔️ 选中。根据 fastboot 使用说明，刷入（flash）子命令格式为 flash PARTITION [FILENAME]。
  - 5165546 fastboot boot &lt;内核镜像文件名或路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ❌ 不选。子命令 boot 用于加载指定的镜像（通常是内核镜像）到设备的内存中用来临时启动，但不会将其实际刷写进设备的 boot 分区。这个命令通常用于测试和开发目的。
- 1574509 **下列说法不正确的是** 【多选题】
  - 5165547 刷入的系统版本与原系统差异较大时，通常建议执行四清
    - ❌ 不选。当刷入系统差异较大时，如果不完整清除数据，可能会无法开机。这里的四清指的是用户数据（Userdata 或 Data）、加密元数据（Metadata）、系统缓存（Cache）以及账户锁（Frp）。
  - 5165548 在计算机的文件系统中，一般采用树型目录结构，在树型目录结构中，树的根结点为根目录，数据文件作为树的结点
    - ✔️ 选中。在树型目录结构中，树的根结点为根目录，数据文件作为树叶，其他所有目录均作为树的结点。
  - 5165549 LRU算法是指当需要淘汰一个页面时，选择驻留主存时间最长的页面进行淘汰
    - ✔️ 选中。选择驻留主存时间最长的页面进行淘汰的算法是 FIFO（First In First Out, 先进先出）。LRU（Least Recently Used, 最近最少使用）算法选择最近最久未使用的页面予以淘汰。该算法赋予每个页面一个访问字段，用来记录一个页面自上次被访问以来所经历的时间 t，当须淘汰一个页面时，选择现有页面中其 t 值最大的，即最近最少使用的页面予以淘汰。
  - 5165550 中断技术使多道批处理系统和I/O设备可与CPU并行工作
    - ❌ 不选。批处理系统分为单道批处理系统和多道批处理系统，中断技术使得多道批处理系统的 I/O 设备可与 CPU 并行工作。
  - 5165551 中继器、路由器、集线器不能抑制广播风暴
    - ✔️ 选中。路由器可以有选择性地转发数据包，可以抑制广播风暴；而中继器和集线器只能广播转发。
- 1574510 **下列说法正确的是** 【多选题】
  - 5165552 VLAN使用的802.1Q帧的最大长度为1522字节
    - ✔️ 选中。802.1Q 是一个用于提供虚拟局域网（Virtual Local Area Network, VLAN）标识和服务质量（Quality of Service, QoS）级别的 IEEE（Institute of Electrical and Electronics Engineers, 国际电气与电子工程师协会） 标准。802.1Q 帧在以太网帧中增加了四个字节，将最大帧大小从 1518 字节增加到 1522 字节。
  - 5165553 给路由器的接口配置好IP地址和地址掩码后，路由器会自动得出该接口的直连网络地址
    - ✔️ 选中。将 IPv4 地址与相应的子网掩码进行逐比特的逻辑与运算，就可得到该 IPv4 地址所在子网的网络地址。
  - 5165554 在一条带宽为4kHz的无噪声通信链路上，若采用8个相位的PSK调制，则该通信链路的最大数据传输速率是24kbps
    - ✔️ 选中。见图片解析：
    <div align="center"><img src = "https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/6e39cba3-054a-44bb-97d4-db18965367fe" width="81.6%" /></div>
  - 5165555 TCP向其上层提供面向连接的可靠传输服务
    - ✔️ 选中。该描述是正确的。
  - 5165556 UDP向其上层提供无连接、不可靠传输服务
    - ✔️ 选中。该描述是正确的。
- 1574511 **下列说法不正确的是** 【多选题】
  - 5165557 TDM的全称是Time Division Multiplexing，是指所有用户在不同的时间占用同样的频带宽度
    - ❌ 不选。时分复用（TDM, Time-division multiplexing）就是将提供给整个信道传输信息的时间划分成若干时间片（简称时隙），并将这些时隙分配给每一个信号源（用户）使用，保证资源的利用率。
  - 5165558 除源站和目的站以外的其他各站，在收到CTS帧或数据帧后，就推迟接入到无线局域网中，保证了源站和目的站之间的通信不会受到其他站的干扰
    - ❌ 不选。本选项是对 CSMA/CA（Carrier Sense Multiple Access with Collision Avoid, 带有冲突避免的载波侦听多路访问）协议中有关“冲突检测”的正确描述。
  - 5165559 虚拟存储容量只受内存容量的限制
    - ✔️ 选中。虚拟存储容量只受计算机地址位数的限制。虚拟存储器的容量永远也不可能超出计算机所能管理的地址空间。
  - 5165560 若一棵二叉树的前序遍历序列为t,e,w,n,b，后序遍历序列为w,b,n,e,t，则根节点的孩子节点只有e
    - ❌ 不选。本选项考查二叉树的遍历。序遍历二叉树先遍历左子树，再遍历右子树，最后访问根结点。前序遍历二叉树先访问根结点，再遍历左子树，最后遍历右子树。已知树的前序遍历和后序遍历并不能唯一的确定一棵树，因此本选项需对各种情况进行分析。前序遍历可以确定根结点必为 t，且必有一个孩子结点 e。假设 t 的孩子结点还有 w，则根据前序遍历可知，e 必为左孩子、w 必为右孩子。此时树的后序遍历为 (()e)(()w)t，也就是说后序遍历序列中的倒数第二个字母一定为 w，而题目给定后序遍历序列中的倒数第二个字母为 e，故该假设不成立。综上，根节点的孩子节点只有 e。
  - 5165561 BGP协议中，open分组用来与相邻的另一个BGP发言人建立关系
    - ❌ 不选。该描述是正确的。BGP（Border Gateway Protocol, 边界网关协议）协议中，open 分组（打开分组）是 TCP 连接建立后发送的第一个消息，用于建立 BGP 对等体（发言人, Speaker）之间的连接关系。
  - 5165562 C语言中的每条可执行语句和非执行语句最终都将被转换成二进制的机器指令
    - ✔️ 选中。C 语言中的非执行语句不会被编译，不会生成二进制的机器指令。
- 1574512 **下列说法正确的是** 【多选题】
  - 5165563 系统总线的位宽指的是总线能同时传送的二进制数据的位数，或数据总线的位数，即32位、64位等总线宽度的概念。总线的位宽越宽，每秒钟数据传输率越大，总线的带宽越宽
    - ✔️ 选中。该描述是正确的。
  - 5165564 I2C也可以叫IIC，是两线式串行通信总线，用于设备间的通讯
    - ✔️ 选中。该描述是正确的。
  - 5165565 I2C通讯只需要两根线，一根是数据线SDA，一根是时钟线SCL
    - ✔️ 选中。该描述是正确的。
  - 5165566 I2C是多主从架构，每个设备都有唯一的地址，一个主设备理论上可以接128个从设备
    - ❌ 不选。I2C 总线最多可以挂多少个设备由 I2C 地址决定，8 位地址，减去 1 位广播地址，是 7 位地址，2^7=128，但是地址 0x00 不用，那就是 127 个地址， 所以理论上可以挂 127 个从器件。
  - 5165567 I2C数据的传输有开始和停止条件，中间进行8bit的数据传输，不校验奇偶，当数据线从低电平转为高电平时，会形成一个开始条件
    - ❌ 不选。发送器件发送一个字节（8bit）+ 校验位，校验位通过接收端数据线实现。I2C 的开始条件是：当“时钟线”为“高电平”，且“数据线”从“高电平”转为“低电平”时。
- 1574513 **下列说法不正确的是** 【多选题】
  - 5165568 汉字的拼音输入码属于汉字的内码
    - ✔️ 选中。汉字的输入码属于外码。不同的输入方法，形成了不同的汉字外码（包括拼音输入码、五笔输入码等等）。
  - 5165569 关系数据库中的数据逻辑结构是树形结构
    - ✔️ 选中。在关系数据库中，存放在数据库中的数据的逻辑结构以二维表为主。
  - 5165570 64进制数3aGC转换为二进制数为11001010101010100110
    - ❌ 不选。64 进制的 `A-Z` 代表 10 进制的 `0-25`，`a-z` 代表 `26-51`，`0-9` 代表 `52-61`，`+/`代表`62-63`。所以 $$3aGC_{(64)}=55\times64^3+26\times64^2+6\times64^1+2\times64^0=830118_{(10)}=2^{19}+2^{18}+2^{15}+2^{13}+2^{11}+2^{9}+2^{7}+2^{5}+2^{2}+2^{1}=11001010101010100110_{(2)}$$
  - 5165571 字长为6位的二进制无符号整数，其最大值是十进制数63
    - ❌ 不选。字长为 6 位的二进制无符号整数，其最大值是十进制数 $2^6-1=63$。
  - 5165572 SPOOLing需要外存的支持
    - ❌ 不选。SPOOLing（Simultaneous Peripheral Operations On-Line, 外部设备联机并行操作）技术通常也叫做假脱机技术，能让多个作业共享一台独占设备，因此其必须建立在具有多道程序功能的操作系统上，而且还应有高速随机外存的支持。
- 1574514 **以下不是用于进行网络异常状态监测和诊断的是** 【多选题】
  - 5165573 SNMP
    - ✔️ 选中。SNMP（Simple Network Management Protocol, 简单网络管理协议）工作在应用层。它是专门设计用于在 IP 网络管理网络节点（服务器、工作站、路由器、交换机及 Hubs 等）的一种标准协议。SNMP 使网络管理员能够管理网络效能，发现并解决网络问题以及规划网络增长。
  - 5165574 ICMP
    - ❌ 不选。ICMP（Internet Control Message Protocol, 网络控制报文协议）工作在网络层。它主要被用于在主机与路由器之间传递控制信息，包括报告错误、交换受限控制和状态信息等。当遇到 IP 数据无法访问目标、IP 路由器无法按当前的传输速率转发数据包等情况时，会自动发送 ICMP 消息。
  - 5165575 UDP
    - ✔️ 选中。UDP（User Datagram Protocol, 用户数据报协议）工作在传输层。它是 OSI（Open System Interconnection, 开放式系统互联）参考模型中的一种无连接的传输层协议，提供面向事务的、简单的、不可靠的信息传送服务，适合于强调传输性能、而不是强调传输完整性的场景。
  - 5165576 ARP
    - ✔️ 选中。ARP（Address Resolution Protocol, 地址解析协议）是一种工作在数据链路层和网络层之间的协议，被用于在计算机网络中解决 IP 地址（属于网络层）与 MAC（Media Access Control）地址（属于数据链路层）之间的转换和映射问题。ARP 协议通过解析目标 IP 地址，从而获得目标 MAC 地址，以便实现网络通信。
  - 5165577 TCPIP
    - ✔️ 选中。TCP/IP（Transmission Control Protocol/Internet Protocol, 传输控制协议/网际协议）工作在传输层。和 UDP 不同的地方在于，TCP 提供面向连接的、稳定的、可靠的传输服务，恰好适应了世界范围内数据通信的需要。
- 1574515 **下列说法不正确的是** 【多选题】
  - 5165578 虚拟局域网只是局域网给用户提供的一种服务，而不是一种新型局域网
    - ❌ 不选。该描述是正确的。
  - 5165579 EDL刷机仅适用于小米14系列和Redmi K70系列之前发布的手机
    - ✔️ 选中。Xiaomi/Redmi 所有搭载高通（Qualcomm）处理器的手机均可使用 Qualcomm Emergency Download（EDL, 9008 模式）刷机。自小米 8 后，小米公司不再允许用户自行使用 EDL 刷机，用户需前往官方售后服务网点才能使用 EDL 刷机。
  - 5165580 Android系统使用TEE来保护极度敏感的数据，解锁BootLoader后可能导致TEE熔断
    - ❌ 不选。小米为了保护用户敏感数据，添加了 TEE 熔断机制，解锁 BootLoader 后，一些刷机行为会触发熔断机制，具体机制介绍可参考本项目维护者的酷安动态。其他厂商出于保护用户数据安全考量，也可能会添加熔断机制（如 Motorola）。
  - 5165581 假脱机的意思是外部设备同时联机操作，采用一组程序或进程模拟一台输入/输出处理器
    - ❌ 不选。该描述是正确的。
  - 5165582 TDM所有用户可以在同样的时间使用同样的频带进行通信
    - ✔️ 选中。时分复用（TDM, Time-division multiplexing）就是将提供给整个信道传输信息的时间划分成若干时间片，在同一时间片内所有用户必须使用不同的频带进行通信，否则将会造成冲突。
  - 5165583 如果网络的传输速率是108.5Kbps，传输27.9M的数据需要约30分钟
    - ✔️ 选中。`27.9MB = 27.9*8Mb = 223.2Mb = 223.2*1024Kb = 228556.8Kb，228556.8Kb/108.5Kbps = 2,106.51s = 35.1min`。
- 1574516 **下列说法不正确的是** 【多选题】
  - 5165584 如果本地域名服务器无缓存，当采用递归法解析另一个网络的某主机域名时，本地域名服务器发送的域名请求消息有1条
    - ❌ 不选。所谓递归查询方式就是：如果主机所询问的本地域名服务器不知道被查询域名的 IP 地址，那么本地域名服务器就以 DNS 客户的身份向其他服务器继续发出查询请求报文，而不是让该主机自行下一步的查询。所以，主机只需向本地域名服务器发送一条域名请求。采用递归查询方法，本地域名服务器也只需向上一级的根域名服务器发送一条域名请求，然后依次递归。
  - 5165585 用户代理和邮件传输代理构成电子邮件系统，常见的邮件协议有SMTP、IMTP、POP3等
    - ✔️ 选中。常见的邮件协议有 SMTP、IMAP、POP3。IMTP 不知道是什么协议，有可能是米米故意改成这样的。
  - 5165586 某网络在物理层规定，信号的电平用+10V~+15V表示二进制0，用-10V~-15V表示二进制1，电线长度限于15m以内，这种物理层接口特性是电气特性
    - ❌ 不选。该描述是正确的。
  - 5165587 管程用于实现进程互斥和进程同步，但管程中定义的变量仅能被管程内的过程访问
    - ❌ 不选。该描述是正确的。
  - 5165588 类加载器的顺序是，先是Bootstrap ClassLoader，然后是Examination ClassLoader，最后是System ClassLoader
    - ✔️ 选中。Java 类加载器的顺序是 Bootstrap ClassLoader -> Extension ClassLoader（而不是 Examination ClassLoader） -> System ClassLoader。
  - 5165589 Trap处理完成后不会返回到Trap指令的下一条指令执行
    - ✔️ 选中。一般来说，Trap（陷阱）处理完成后，会继续返回到 Trap 指令的下一条指令继续执行。但当 Trap 指令是转移指令时，就不是返回到下一条指令执行了，而是返回到转移目标指令执行。
- 1574517 **在TCP/IP参考模型中，不是由传输层相邻的下一层实现的主要功能是** 【多选题】
  - 5165590 对话管理
    - ✔️ 选中。对话管理是由 OSI 参考模型中的「会话层」所实现的功能。
  - 5165591 路由选择
    - ❌ 不选。TCP/IP 模型中，传输层相邻的下一层是网络层，网络层所实现的功能是路由选择和分组转发。
  - 5165592 端到端报文段传输
    - ✔️ 选中。端到端报文段传输是由 TCP/IP 模型中的「传输层」所实现的功能。
  - 5165593 结点到结点流量控制
    - ✔️ 选中。结点到结点流量控制是由 TCP/IP 模型中的「数据链路层」所实现的功能。
- 1574518-1 **请截图本页面，并上传截图（此题根据答题时间情况来判断您实际填空题用时，填空题用时过短会被判为无效答卷）** 【文件题】
  - 0 [截图识别码1099-1]未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ✔️ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。
- 1574518-2 **请截图本页面，并上传截图（此题根据答题时间情况来判断您实际填空题用时，填空题用时过短会被判为无效答卷）** 【文件题】
  - 0 [截图识别码2341-3]未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ✔️ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。
- 1574519 **以下是BL解锁高频申请理由，请分析该理由不通过可能的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“尊敬的审核人员，我想申请解锁我的手机 bootloader，因为我希望能够完全掌控自己的设备。我相信这是我的权利，也可以提高我对手机的使用体验。解锁 bootloader 是一个简单而有效的方式，可以让我使用更多的自定义选项和优化，从而帮助我更好地管理我的手机。除此之外，我也是一名开发者和技术爱好者，我相信通过解锁 bootloader，我可以更深入地了解...”
    - ✔️ 至少需要输入 30 个字符。
    （1）该申请理由疑似使用 AI 生成，根据米米在本问卷第 2 题中的第 3 个选项提示可知，使用 AI 生成申请理由是不被允许的；
    （2）BL 解锁后用户确实可以拥有对设备更高的控制权，但是也会带来更多的安全风险；
    （3）解锁 BL 并不一定能带来更好的使用体验，根据米米在本问卷第 4 题的提示，解锁 BL 带来的危害（手机变砖、信息泄露、各类 App 无法正常使用等）要大于其带来的便利（自定义选项和优化）；
    （4）“相信通过解锁 BootLoader，我可以更深入地了解”这句话说明，该用户的玩机经验可能不足，为了保护其免于遭受解锁后的各类风险，其申请不予通过。
- 1574520 **以下是BL解锁高频申请理由，请分析该理由不通过的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“尊敬的小米公司：您好！我是一名小米手机用户，最近我购买了一款小米手机，在使用过程中，我发现了一些特别的功能和特点。因此，我希望能够申请解锁手机的BL（Bootloader），以便更好地了解和使用这些功能。”
    - ✔️ 至少需要输入 30 个字符。
    （1）该申请理由疑似使用 AI 生成，根据米米在本问卷第 2 题中的第 3 个选项提示可知，使用 AI 生成申请理由是不被允许的；
    （2）该用户并没有具体说明是什么特别的功能和特点，使用这些功能又和 BL 解锁有什么关系，表述非常模糊笼统；
    （3）对于普通用户而言，目前 HyperOS 并不存在需要 BL 解锁才能体验到的功能特性，因此该申请理由不够充分；
    （4）该用户只看到了 BL 解锁后可能带来的新特性，对 BL 解锁带来的风险却没有具体的认知，故其申请不能被通过。
- 1574521 **以下是BL解锁高频申请理由，请分析该理由不通过的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“愿意尝试新事物，有毅力和精力进行测试，并能随时报告测试情况。具备测试和反馈的能力，能够分析系统存在的问题并提供详细的反馈报告。精通MIUI系统的使用和问题解决，MIUI系统是小米的操作系统，熟悉其风格和功能。”
    - ✔️ 至少需要输入 30 个字符。
    （1）该申请理由疑似直接套用其他内测项目（如开发版）的申请理由，不适用于 BL 解锁申请；
    （2）申请 BL 解锁不属于“尝试新事物”的范畴，因为 BL 解锁带来的风险远大于普通的内测项目；
    （3）BL 解锁并不属于测试，不存在报告或反馈测试情况的说法，分析系统问题、提供详细反馈属于开发版和稳定版内测的范畴，而不是 BL 解锁；
    （4）“精通 MIUI 系统的使用”，MIUI 系统解锁 BL 并不需要申请该项目，只有需要解锁 BL 的 HyperOS 用户才适用该项目的申请，故该申请不予通过。
- 1574522 **以下是BL解锁高频申请理由，请分析该理由不通过的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“我迫切希望能够解锁BL，以便能够更加深入地了解和体验澎湃OS的强大功能，解锁BL后，我可以更好地参与到系统的优化和完善过程中，为澎湃OS的改进和发展贡献”
    - ✔️ 至少需要输入 30 个字符。
    （1）该申请理由疑似直接套用其他内测项目（如开发版）的申请理由，不适用于 BL 解锁申请；
    （2）解锁 BL 与体验 HyperOS 的强大功能没有关系，对于普通用户而言，目前 HyperOS 并不存在需要 BL 解锁才能体验到的功能特性；
    （3）如果希望“参与到系统的优化和完善过程”当中，该用户应该申请开发版和稳定版内测，而不是申请 BL 解锁；
    （4）“为澎湃 OS 的改进和发展贡献”和申请 BL 解锁没有关联，该申请理由全文实际上都 BL 解锁无关，故不能被通过。
- 1574523 **如您在其他技术交流平台有较多贡献（如：GitHub等），可将相关信息进行填写，并在下一题上传相关截图凭证** 【填空题】
  - 0 非必填，此题的有效填写会提升审核通过率
    - ✔️ 非必填。「？」
- 1574524 **请将上一题填写的其他技术交流平台贡献信息，上传相关截图凭证，以供审核使用** 【文件题】
  - 0 非必填，此题的有效填写会提升审核通过率
    - ✔️ 非必填。如您上一题填写了相关内容，请于本题上传截图凭证。

**Reply by K1anaKaslana** [reply]
笑死，真的ex人

**Reply by Buzaijun** [reply]
46分，又要等下周咯，填空题时间太短了，根本来不及打完这么多字

**Reply by gugeplay** [reply]
放心就算你是手写的也会判定为复制粘贴的

**Reply by Jerry-FaGe** [reply]
感觉以后求职，也不需要啥面试笔试了；只要跟人说我能15分钟独立通关小米最新解锁试题直接入职，贵司捡到宝了

**Reply by ANCWzhang** [reply]
46分直接拜拜

**Reply by Nomi7** [reply]
46分？？？哦大家好像都是46分。。。

**Reply by smallxjj** [reply]
笑死，46分，谁写的答案？

**Reply by xbn2002** [reply]
> 46分直接拜拜

今早题没变吧，截图题的截图还有留嘛，有的话发一下，我看看识别码变了没

**Reply by Nomi7** [reply]
![微信图片_20240410085318](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/50700531/f713cfa8-add2-4ed0-a373-492979964e20)

**Reply by xbn2002** [reply]
好的，谢谢，和昨天下午场一样，没变

**Reply by PDxtXHd** [reply]
> 笑死，46分，谁写的答案？

这种情况我们一般建议下次自己答

**Reply by Edwin-Kevin** [reply]
> 笑死，46分，谁写的答案？

你好，下次可以自己独立作答的。

**Comment by xbn2002** [comment]
2024-04-09 14:45-15:00
本场题目除了截图识别码有变更外，和 [2024-04-09 08:00-08:15](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/173#discussioncomment-9045817) 一致
- 1574518 **请截图本页面，并上传截图（此题根据答题时间情况来判断您实际填空题用时，填空题用时过短会被判为无效答卷）** 【文件题】
  - 0 [截图识别码2341-3]未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ✔️ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。

**Reply by yixiancc** [reply]
用chatgpt都只拿到了22分，我也是服了

**Comment by xbn2002** [comment]
2024-04-10 08:00-08:15
本场题目和 [2024-04-09 14:45-15:00](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/discussions/173#discussioncomment-9045818) 一致

**Comment by MiFanLover1** [comment]
内鬼开设的 xiaomikey 秒解服务真是狮子大开口，得寸进尺，今天一看居然已经要价 95 刀，都快 700 人民币了，这尼玛解毛线啊，社区又不给过，小米 tm 是 nt
<div align="center"><img src="https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/2c54c56f-71c2-4938-8f39-fd1a823ca715" width="350px"></div>
<table align="center">
    <tr>
        <th>日期</th>
        <th>美元价格</th>
        <th>换算人民币价格</th>
    </tr>
    <tr>
        <td>2024-06-03</td>
        <td>USD $120.00</td>
        <th>RMB ¥869.45</th>
    </tr>
    <tr>
        <td>2024-04-27</td>
        <td>USD $100.00</td>
        <th>RMB ¥724.54</th>
    </tr>
    <tr>
        <td>2024-04-08</td>
        <td>USD $95.00</td>
        <th>RMB ¥688.31</th>
    </tr>
    <tr>
        <td>2024-03-21</td>
        <td>USD $75.00</td>
        <th>RMB ¥543.41</th>
    </tr>
    <tr>
        <td>2024-02-18</td>
        <td>USD $65.00</td>
        <th>RMB ¥470.95</th>
    </tr>
    <tr>
        <td>2024-02-13</td>
        <td>USD $50.00</td>
        <th>RMB ¥362.27</th>
    </tr>
    <tr>
        <td>2024-01-27</td>
        <td>USD $61.99</td>
        <th>RMB ¥449.14</th>
    </tr>
    <tr>
        <td>2024-01-26</td>
        <td>USD $55.00</td>
        <th>RMB ¥398.50</th>
    </tr>
    <tr>
        <td>2024-01-23</td>
        <td>USD $38.50</td>
        <th>RMB ¥278.95</th>
    </tr>
    <tr>
        <td>2023-11-30</td>
        <td>USD $27.99</td>
        <th>RMB ¥202.80</th>
    </tr>
</table>

**Reply by chushuangyl** [reply]
难道是通过国际版解锁？

**Reply by mufengran** [reply]
内鬼授权重定向远程解
